content/blog/posts/visual-studio-2022-update.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "O3DE Visual Studio 2022 Support!"
+date: 2022-11-09
+slug: o3de-vs2022-initial-support
+author: lumberyard-employee-dm
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: ""
+---
+With the 22.10.0 release of the **Open 3D Engine(O3DE)** Visual Studio 2022(VS2022) is now fully supported within the O3DE.
+
+## Background
+
+When we say that **O3DE** "fully supports" Visual Studio 2022, we are referring to building O3DE projects using Visual Studio 2022 [v143 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset).  
+![Visual Studio 2022 v143 toolset](/images/blog/vs2022/vs2022-installer-v143-toolset.png)
```

Comment:
`![Visual Studio 2022 v143 toolset](/images/blog/vs2022/vs2022-installer-v143-toolset.png)`
Change to
`{{< image-width src="/images/blog/vs2022/vs2022-installer-v143-toolset.png" width="100%" alt="Visual Studio 2022 v143 toolset" >}}`